src/test/java/seedu/address/model/tag/TagTest.java

```diff
@@ -18,9 +20,54 @@ public void constructor_invalidTagName_throwsIllegalArgumentException() {
     }
 
     @Test
-    public void isValidTagName() {
-        // null tag name
+    public void isValidTagName_null_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> Tag.isValidTagName(null));
     }
 
+    @Test
```

Comment:
Should the test class be named more specifically since we will have test classes such as TagCommandTest and NewtagCommandTest?

Reply:
I think it's fine since we can stick to the convention of adding "Test" behind the original class name for test classes!